runtime/queries/metricsview.go

```diff
@@ -26,6 +26,8 @@ import (
 	"google.golang.org/protobuf/types/known/structpb"
 )
 
+var emptyMeasureAliases = make([]*runtimev1.MetricsViewComparisonMeasureAlias, 0)
```

Comment:
Not needed, can just pass `nil` because `range` and `len` are valid on a `nil` slice or map.